platform/qt/src/qmapboxgl.cpp

```diff
@@ -471,26 +668,52 @@ void QMapboxGL::setGestureInProgress(bool inProgress)
     d_ptr->mapObj->setGestureInProgress(inProgress);
 }
 
+/*!
+    Adds an \a className to the list of active classes. Classes are like filters, layers
+    tagged with a certain class will only be active when the class is added.
```

Comment:
It might be worth mentioning core support for classes are deprecated, as per comments in [the specification](https://www.mapbox.com/mapbox-gl-style-spec/#layer-paint.*).